docs/emulator-suite/_includes/_functions_instrument.md

```diff
@@ -0,0 +1,47 @@
+### Callable Functions
```

Comment:
Extracted into its own file. We can modify devsite to include this file as well. No actual changes.